src/gdcmodels/export/graph/case.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+from gdcmodels import esmodels
+from gdcmodels.export.graph import common
+
+
+class Exporter(common.GraphExporter):
+    __slots__ = ("_cases", "_files")
+
+    def __init__(
+        self,
+        cases: Optional[common.CaseProperties] = None,
+        files: Optional[common.FileProperties] = None,
+        gdc_dictionary: Optional[common.GDCDictionary] = None,
+    ) -> None:
+        super().__init__(gdc_dictionary)
+
+        self._cases = cases or common.CaseProperties()
+        self._files = files or common.FileProperties()
+
+    def _get_file_properties(self) -> common.NestedDict:
+        properties = self._files.export_properties()
+
+        del properties["annotations"]
+        del properties["associated_entities"]
+
+        return properties
+
+    def _export_mapping(self) -> esmodels.ESMapping:
+        mapping: esmodels.ESMapping = {
+            "properties": {
+                **self._cases.export_properties(),
+                "files": esmodels.Property(properties=self._get_file_properties(), type="nested"),
+            }
+        }
+
+        self._add_autocomplete(
+            mapping,
+            "case_autocomplete",
+            (
+                "case_id",
```

Comment:
Looks like a candidate for external configuration.

Reply:
Not sure I'm following. Where externally would you want this configured?

Reply:
I'm trying to think through what a developer would be doing when they update the models. They are going to update the mapping and vestigial.yaml files. Would it make sense to drop in an autocomplete.yaml next to those?

Reply:
For the graph indices, we shouldn't be touching the mapping nor the vestigial yaml files. They are completely generated from scratch in this script as, at least the mappings, were generated in esbuild (before we "broke" things by deleting fields.) 

If you look at the sync process [here](https://github.com/NCI-GDC/gdc-models/blob/8bb9557017573d202d4e11505a3514c7a1b5c7f2/src/gdcmodels/sync/graph/common.py#L278-L282), you'll see that for the graph that it just resets the mappings completely to what is generated. So if a dev were to try and manually update a mapping field that would not be generated by the graph sync logic:
A) if they didn't then run the sync process, [this](https://github.com/NCI-GDC/gdc-models/blob/8bb9557017573d202d4e11505a3514c7a1b5c7f2/tests/test_sync.py#L23-L36) test would fail as it insures that the synced mappings are the same as those loaded by `get_es_models`.
B) their newly added fields would instantly appear in the vestigial properties after sync.

Reply:
I'm glad to hear that mapping and vestigal.yaml files are generated. The 'auto_complete' configuration still needs to be manually editing in the source code. Is there any indicators in the dictionary to drive this value? Or is it unique for each?

Reply:
As far as I know, the business drives what is included in the autocomplete feature.